libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -253,14 +253,36 @@ private IdentityClaim getIdentityClaim(
 
     private Optional<IdentityClaim> generateIdentityClaim(List<VcStoreItem> successfulVCStoreItems)
             throws HttpResponseExceptionWithErrorBody {
+        if (successfulVCStoreItems.isEmpty()) {
+            LOGGER.warn("Failed to generate identity claim");
+            return Optional.empty();
+        }
+
+        List<IdentityClaim> identityClaims = new ArrayList<>();
         for (VcStoreItem item : successfulVCStoreItems) {
             if (EVIDENCE_CRI_TYPES.contains(item.getCredentialIssuer())) {
-                return Optional.of(
-                        getIdentityClaim(item.getCredential(), item.getCredentialIssuer(), false));
+                identityClaims.add(
+                        getIdentityClaim(item.getCredential(), item.getCredentialIssuer(), true));
             }
```

Comment:
I think it'd be really good if we could get rid of `EVIDENCE_CRI_TYPES` completely. At the moment it contains `PASSPORT_CRI, DCMAW_CRI, DRIVING_LICENCE_CRI, F2F_CRI`. We've agreed not to add BAV and NINO to it, so it's already a weirdly inconsistent thing.
One possibility it to check if the `VcStoreItem` contains an evidence and validity score, that way we know it's an evidence VC and can be used to build the identity claim.

Reply:
I've removed this in favor of a new `isEvidenceVc` method